README-aria.md

```diff
@@ -0,0 +1,71 @@
+# ARIA PR Notes:
```

Comment:
Not sure we want another file in the root for this. If its is good info put in the normal README.md

Reply:
agreed. will do